<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QModbus_RTU_MASTER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Modbus parametr</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="5">
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>Czytaj ZmiennÄ…</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="LE_Mb_adress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="CB_Reg_count"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="LE_Slave_adress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Slave Adress</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="CB_MB_function">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Adress Modbus</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="CB_Adress_offset"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Offset Adresu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_43">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Status Program</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1" rowspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioCom">
         <property name="text">
          <string> Port Com Aktywny</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioEth">
         <property name="text">
          <string>Ethernet Aktywny</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ramka Odpowiedzi Modbus (HEX)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QLineEdit" name="lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Interpretacja Zmiennych</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_21">
         <property name="title">
          <string>UINT32 ABCD</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_45">
          <item>
           <widget class="QLineEdit" name="LE_UINT32_ABCD">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>UINT BA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="LE_UINT_BA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QGroupBox" name="groupBox_18">
         <property name="title">
          <string>UINT46 HGFEDCBA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLineEdit" name="LE_UINT64_DCBA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>INT32 CDAB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="LE_INT32_CDAB">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_15">
         <property name="title">
          <string>INT64 ABCDEFGH</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLineEdit" name="LE_INT64_ABCD">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>INT32 ABCD</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="LE_INT32_ABCD">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QGroupBox" name="groupBox_14">
         <property name="title">
          <string>FLOAT BADC</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLineEdit" name="LE_FLOAT_BADC">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QGroupBox" name="groupBox_24">
         <property name="title">
          <string>UINT32 BADC</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_48">
          <item>
           <widget class="QLineEdit" name="LE_UINT32_BADC">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="groupBox_19">
         <property name="title">
          <string>DOUBLE_64 ABCDEFGH</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLineEdit" name="LE_DOUBLE_ABCD">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>FLOAT ABCD</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLineEdit" name="LE_FLOAT_ABCD">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>INT32 DCBA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="LE_INT32_DCBA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>FLOAT DCBA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLineEdit" name="LE_FLOAT_DCBA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QGroupBox" name="groupBox_23">
         <property name="title">
          <string>UINT32 CDAB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_47">
          <item>
           <widget class="QLineEdit" name="LE_UINT32_CDAB">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>INT AB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="LE_INT_AB">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QGroupBox" name="groupBox_13">
         <property name="title">
          <string>FLOAT CDAB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLineEdit" name="LE_FLOAT_CDAB">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QGroupBox" name="groupBox_17">
         <property name="title">
          <string>UINT64 ABCDEFGH</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLineEdit" name="LE_UINT64_ABCD">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>UINT AB</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="LE_UINT_AB">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>INT32 BADC</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="LE_INT32_BADC">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QGroupBox" name="groupBox_16">
         <property name="title">
          <string>INT64 HGFEDCBA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLineEdit" name="LE_INT64_DCBA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>INT BA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="LE_INT_BA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_22">
         <property name="title">
          <string>UINT32 DCBA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_46">
          <item>
           <widget class="QLineEdit" name="LE_UINT32_DCBA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QGroupBox" name="groupBox_20">
         <property name="title">
          <string>DOUBLE_64 HGFEDCBA</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLineEdit" name="LE_DOUBLE_DCBA">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>UrzÄ…dzenia</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRTU_OVER_TCP"/>
    <addaction name="actionRTU_COM_PORT"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRTU_OVER_TCP">
   <property name="text">
    <string>Ustawienia Ethernet</string>
   </property>
  </action>
  <action name="actionRTU_COM_PORT">
   <property name="text">
    <string>Ustawienia COM</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
